core/templates/pages/admin-page/jobs-tab/admin-jobs-tab.directive.ts

```diff
@@ -41,79 +40,70 @@ angular.module('oppia').directive('adminJobsTab', [
       controller: [function() {
         var ctrl = this;
         ctrl.showJobOutput = function(jobId) {
-          var adminJobOutputUrl = UrlInterpolationService.interpolateUrl(
-            ADMIN_JOB_OUTPUT_URL_TEMPLATE, {
-              jobId: jobId
+          AdminBackendApiService.showJobOutputAsync(jobId)
+            .then(function(response) {
+              ctrl.showingJobOutput = true;
+              ctrl.jobOutput = response || [];
```

Comment:
Handle this in the service file i.e, return an empty list if there's no output.

Reply:
done